content/applications/essentials/contacts.rst

```diff
@@ -220,16 +215,24 @@ drop-down menu. Then, configure a :guilabel:`Member Price`. Click :guilabel:`Inv
 when both fields are filled in.
 
 Alternatively, to offer a free membership, tick the :guilabel:`Free Member` checkbox, in the
-:guilabel;`Membership` tab of a contact form.
+:guilabel:`Membership` tab of a contact form.
+
+Publish members directory
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To publish a list of active members on the website, the *Online Members Directory* application must
+first be installed. Navigate to the :menuselection:`Apps application` and remove the
+:guilabel:`Apps` filter from the :guilabel:`Search...` bar. Then, search for `Online Members
+Directory`.
+
+Click :guilabel:`Install` on :guilabel:`Online Members Directory` card that appears. Doing so
+installs the module and returns to the main Odoo dashboard.
```

Comment:
I dont think it is necessary to explain how to install the module, can just reference the existing documentation.

Then make sure any referenced documentation is contextual, rather than the vague "these steps" :)

Also, detail the `/members` page URL.
![members-menu-item](https://github.com/odoo/documentation/assets/36018073/75749247-d609-4535-8df7-b8e37a74f884)

```suggestion
To publish a list of active members on the website, the *Online Members Directory* application must
first be :ref:`installed <general/install>`. After installing the module, add the `/members` page to
the website's menu by :doc:`editing the website menu <../websites/website/pages/menus>`.
```